File: flutterapp/lib/recipe_name.dart
import 'package:flutter/material.dart';

import 'theme.dart';

class RecipeNameWidget extends StatelessWidget {
  const RecipeNameWidget({
    Key? key,
  }) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Text(
      'Creamy Pesto Pasta',
      style:
          Theme.of(context).textTheme.headline3!.copyWith(color: basilOrange),
      textAlign: TextAlign.center,
    );
  }
}

File: composeapp/app/src/main/java/com/example/composeapp/RecipeName.kt
package com.example.composeapp

import androidx.compose.material.MaterialTheme
import androidx.compose.material.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.text.style.TextAlign

@Composable
fun RecipeName() {
    Text(
        text = "Creamy Pesto Pasta",
        style = MaterialTheme.typography.h3.copy(color = basilOrange),
        textAlign = TextAlign.Center
    )
}
